guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -12,6 +12,12 @@ include::snip_firewalld.adoc[]
 ----
 # yum install postgresql-server
```

Comment:
This should match the EVR package, which means `rh-postgresql12-postgresql-server`. You may want to suggest `rh-postgresql12-syspaths`, but that's not strictly needed.

Reply:
Do you say that the command `yum install postgresql-server` doesn't work and should be substituted by `yum install rh-postgresql12-syspaths`?

Reply:
Yes. Effectively you want to install `yum install ${scl_prefix}postgresql-server ${scl_prefix}postgresql-evr`. On EL7 `$scl_prefix` is `rh-postgresql12-` (right now) but on EL8 it would be empty.

If you don't install `rh-postgresql12-syspaths` the service also becomes `rh-postgresql12-postgresql` instead of `postgresql` so `systemctl` commands would also need to be modified. If the syspaths package is installed, you can keep using all the regular paths, but it will conflict with the regular postgresql packages.

Note that if users have an external postgresql they also need to upgrade their installation. https://github.com/theforeman/foreman-installer/blob/71f60592f33a3056f52791a86876b9ae3ce9235f/hooks/pre/30-el7_upgrade_postgresql.rb is what our installer does for systems where it's bundled. Also note that the storage path changed to `/var/opt/rh/rh-postgresql12/lib/pgsql` instead of `/var/lib/pgsql`. This may have impact on storage requirements and backup procedures.

Reply:
I used the packages for EL7 because the guides say to install PostgreSQL on RHEL8.

Reply:
Updated paths and PostgreSQL supported version in #271